File: api/code-samples/python/klay/gas/feeHistory.py
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

blockCount = "0x10"
lastBlock = "latest"
rewardPercentiles = [0.1, 0.2, 0.3]

sdk = OpenSDK(host)
klay_response = sdk.klay.fee_history(blockCount, lastBlock, rewardPercentiles)

print(klay_response)

File: api/code-samples/javascript/klay/gas/feeHistory.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new Caver.ApiClient("https://api.baobab.klaytn.net:8651"));

    const blockCount = '0x10'
    const lastBlock = 'latest'
    const rewardPercentiles = [0.1, 0.2, 0.3]
    sdk.klay.feeHistory(blockCount,lastBlock,rewardPercentiles,{}, (err, data, response) => {
        console.log(data);
    });
   
}
)()